dojo/finding/views.py

```diff
@@ -687,7 +687,7 @@ def copy_finding(request, fid):
                                 title='Copying of %s' % finding.title,
                                 description='The finding "%s" was copied by %s to %s' % (finding.title, request.user, test.title),
                                 product=product,
-                                url=request.build_absolute_uri(reverse('copy_finding', args=(finding_copy.id, ))),
+                                url=request.build_absolute_uri(reverse('finding_copy', args=(finding_copy.unsaved_vulnerability_ids, ))),
```

Comment:
`unsaved_vulnerability_ids` is an array that don't have finding id. I don't think this will work